wormchain/mainnet/genesis.json

```diff
@@ -0,0 +1,351 @@
+{
+  "genesis_time": "2022-09-27T22:08:43.044644477Z",
+  "chain_id": "wormchain",
+  "initial_height": "1",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "22020096",
+      "max_gas": "-1",
+      "time_iota_ms": "1000"
+    },
+    "evidence": {
+      "max_age_num_blocks": "100000",
+      "max_age_duration": "172800000000000",
+      "max_bytes": "1048576"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {}
+  },
+  "app_hash": "",
+  "app_state": {
+    "auth": {
+      "params": {
+        "max_memo_characters": "256",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "address": "wormhole1cm4lqyl5j5t9ewd79vrnsrga2cnml8ca29r0sz",
+          "pub_key": null,
+          "account_number": "0",
+          "sequence": "0"
+        }
+      ]
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
+        {
+          "address": "wormhole1cm4lqyl5j5t9ewd79vrnsrga2cnml8ca29r0sz",
+          "coins": [
+            {
+              "denom": "utest",
+              "amount": "10000"
+            }
+          ]
+        }
+      ],
+      "supply": [],
+      "denom_metadata": [
+        {
+          "description": "Wormchain's native test asset",
+          "denom_units": [
+            {
+              "denom": "utest",
```

Comment:
should there be a test token?

Reply:
Yes I'd like to use the test token to create the guardian accounts when guardians generate their validator addresses. I.e.:
1. guardian generates validator address.
2. we send them a test token as a quick way to create the account for their address.